scripts/portView.js

```diff
@@ -0,0 +1,76 @@
+/*portView.js is a file dedicated to handling the viewing functionality of the portfolio articles and list*/
+
+var portArticleView = {};
+
+portArticleView.handlePreview = function(){
+  $('.portPreview').hide();
+
+  $('#portList').on('click', 'a' , function(e){
+    /*TODO: Add conditional statement to prevent opening another port article if one is already open. */
+    e.preventDefault();
+    var title = $(this).text();
+    $('article[data-title="' + title + '"]').slideToggle();
+  });
+
+  $('article.portPreview').on('click', 'a.icon-minus', function(e){
+    e.preventDefault();
+    $(this).parent().slideToggle();
+  });
+};
+
+portArticleView.handleMainNav = function(){
+  $('.siteLinks').on('click', '.tab', function(e){
+    e.preventDefault();
+
+    var tab = $(this).text().toLowerCase();
```

Comment:
if your only gonna use the variable tab once then just put $(this).text().toLowerCase() in the one play you need it. Unless you want it to know what $(this).text().toLowerCase() is automatically without sifting through the code.